shared/common-adapters/button.desktop.js

```diff
@@ -2,24 +2,38 @@
 
 import React, {Component} from 'react'
 import {FlatButton} from 'material-ui'
-import {globalStyles, globalColors} from '../styles/style-guide'
+import {globalStyles, globalColors, globalColorsDZ2} from '../styles/style-guide'
 import type {Props} from './button'
+import flags from '../util/feature-flags'
 
 export default class Button extends Component {
   props: Props;
 
   render () {
-    const rootStyle = this.props.primary ? styles.buttonPrimary : styles.buttonSeconary
+    let styles
+    if (flags.tracker2) {
+      styles = styles2
+    } else {
+      styles = styles1
+    }
+
+    const rootStyle = (this.props.follow || this.props.primary) ? styles.buttonPrimary : styles.buttonSecondary
     const smallStyle = this.props.small ? styles.buttonSmall : {}
+    const followStyle = this.props.follow ? styles2.buttonFollow : {}
     const smallLabelStyle = this.props.small ? styles.buttonSmallLabel : {}
     const moreStyle = this.props.more ? styles.buttonMore : {}
     const moreLabelStyle = this.props.more ? styles.buttonMoreLabel : {}
 
+    // FIXME: A better way?
```

Comment:
This is all opacity based and not based on the color of the background. I mean the entire thing is 20% opacity (etc) not just the text or background